scripts/run-massif.sh

```
@@ -0,0 +1,80 @@
+# Check if binary exists
+BINARY="${1:-bin/moac}"
+if [ ! -f "$BINARY" ]; then
+    echo "Error: Binary not found: $BINARY"
+    echo ""
+    echo "Usage: $0 [binary] [args...]"
+    echo "Example: $0 bin/moac"
+    exit 1
+fi
+
+shift 2>/dev/null || true
+ARGS="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)